helm-chart-sources/pulsar/values.yaml

```diff
@@ -1035,6 +1035,8 @@ function:
   #   mainContainerStartupCmd: "cp -f /connectors/pulsar-connectors/* /pulsar/connectors"
 
   authenticationProviders: "org.apache.pulsar.broker.authentication.AuthenticationProviderToken,org.apache.pulsar.broker.authentication.AuthenticationProviderTls"
+  # upload false means the connector jar will not uploaded from bookie but from the same version as function worker
+  uploadBuiltinSinksSources: "false"
```

Comment:
I think we should remove this configuration option and set it to `false` always. Unless I'm misunderstanding something, I don't see any reason for a user to set this as `true` when running in k8s. Do you agree @zzzming or do you think we should make this configurable?